socs/agents/srs_cg635m/agent.py

```diff
@@ -0,0 +1,198 @@
+import argparse
+import socket
+import time
+from os import environ
+
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import TimeoutLock
+
+from socs.agents.srs_cg635m.drivers import SRS_CG635m_Interface
+
+
+class SRSCG635mAgent:
```

Comment:
Needs a docstring.

Also, more generally, we need a docs page for this agent in `socs/docs/agents/`. Plenty of examples there, but also see https://ocs.readthedocs.io/en/main/developer/agent_references/documentation.html.

This is especially important for deciphering those returned integers.